scripts/create_policy

```diff
@@ -129,15 +131,46 @@ def process_ima_buf_in_measurement_list(
     return keyrings_map, ima_buf_map, ret
 
 
+def process_signature_verification_keys(verification_keys, policy):
+    """Add the given keys (x509 certificates) to keyring"""
+
+    verification_key_list = None
+    if verification_keys:
+        if policy.get("verification-keys", None):
+            keyring = file_signatures.ImaKeyring().from_string(policy["verification-keys"])
+            if not keyring:
+                print("Could not create IMAKeyring from JSON")
+        else:
+            keyring = file_signatures.ImaKeyring()
+        if keyring:
+            for key in verification_keys:
+                try:
+                    pubkey, keyidv2 = file_signatures.get_pubkey_from_file(key)
+                    if not pubkey:
+                        print(f"File '{key}' is not a file with a key")
+                    else:
+                        keyring.add_pubkey(pubkey, keyidv2)
+                except ValueError as e:
+                    print(f"File '{key}' does not have a supported key: {e}")
+            verification_key_list = keyring.to_string()
+
+    if verification_key_list:
+        policy["verification-keys"] = verification_key_list
+
+    return policy
+
+
 def main(argv):
     """main"""
-    parser = argparse.ArgumentParser(argv[0])
+    parser = argparse.ArgumentParser(
+        prog=argv[0], description="This is an experimental tool for adding items to a Keylime's IMA runtime policy"
```

Comment:
`prog=argv[0]` can be dropped, it is taken by default

Reply:
Thanks. Fixed.